packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -128,6 +131,20 @@ export const useArbTokenBridge = (
     provider: l2.provider,
     walletAddress
   })
+  const {
+    eth: [, updateEthL1CustomDestinationBalance],
+    erc20: [, updateErc20L1CustomDestinationBalance]
+  } = useBalance({
+    provider: l1.provider,
+    walletAddress: destinationAddress
```

Comment:
Would it make sense to do something like:
```
walletAddress: !isDepositMode && isDifferentDestinationAddress ? destinationAddress : walletAddress
```

So we don't have to maintain 2 hooks and call 2 methods? Similarly for L2.